linode/vpcips/framework_schema_datasource.go

```diff
@@ -0,0 +1,91 @@
+package vpcips
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper/frameworkfilter"
+)
+
+var VPCIPAttrs = map[string]schema.Attribute{
+	"address": schema.StringAttribute{
+		Description: "An IPv4 address configured for this VPC interface. These follow the RFC 1918 private address format. Displayed as null if an address_range.",
+		Computed:    true,
+	},
+	"gateway": schema.StringAttribute{
+		Description: "The default gateway for the VPC subnet that the IP or IP range belongs to.",
+		Computed:    true,
+	},
+	"linode_id": schema.Int64Attribute{
+		Description: "The identifier for the Linode the VPC interface currently belongs to.",
+		Computed:    true,
+	},
+	"prefix": schema.Int64Attribute{
+		Description: "The number of bits set in the subnet_mask.",
+		Computed:    true,
+	},
+	"region": schema.StringAttribute{
+		Description: "The region of the VPC.",
+		Computed:    true,
+	},
+	"subnet_mask": schema.StringAttribute{
+		Description: "The mask that separates host bits from network bits for the address or address_range.",
+		Computed:    true,
+	},
+	"nat_1_1": schema.StringAttribute{
+		Description: "The public IP address used for NAT 1:1 with the VPC. This is empty if NAT 1:1 isn't used.",
+		Computed:    true,
+	},
+	"subnet_id": schema.Int64Attribute{
+		Description: "The id of the VPC Subnet for this interface.",
+		Computed:    true,
+	},
+	"config_id": schema.Int64Attribute{
+		Description: "The globally general entity identifier for the Linode configuration profile where the VPC is included.",
+		Computed:    true,
+	},
+	"interface_id": schema.Int64Attribute{
+		Description: "The globally general API entity identifier for the Linode interface.",
+		Computed:    true,
+	},
+	"address_range": schema.StringAttribute{
+		Description: "A range of IPv4 addresses configured for this VPC interface. Displayed as null if a single address.",
+		Computed:    true,
+	},
+	"vpc_id": schema.Int64Attribute{
+		Description: "The unique globally general API entity identifier for the VPC.",
+		Computed:    true,
+	},
+	"active": schema.BoolAttribute{
+		Description: "Returns true if the VPC interface is in use, meaning that the Linode was powered on using the config_id to which the interface belongs. Otherwise returns false",
+		Computed:    true,
+	},
+}
+
+var filterConfig = frameworkfilter.Config{
+	"active":    {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeBool},
+	"config_id": {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeInt},
+	"linode_id": {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeInt},
+	"region":    {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"vpc_id":    {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeInt},
```

Comment:
I think these attributes are API filterable?

Reply:
Thanks, I fixed this! Just for future reference, where can you find if a field is listed as API Filterable?

Reply:
I think the API filterable fields are noted in the API docs?
<img width="291" alt="image" src="https://github.com/user-attachments/assets/70b21d10-1a21-4455-bc90-bc7f689759e5">
https://techdocs.akamai.com/linode-api/reference/get-vpcs-ips

Reply:
Oh that makes sense! So fields with API Filterable set to false are additional filters added to terraform that are not supported at the API level?

Reply:
Yes, I think so